plugin/dns64/README.md

```diff
@@ -0,0 +1,47 @@
+# dns64
+
+The `dns64` plugin implements the DNS64 IPv6 transition mechanism. From Wikipedia:
+
+> DNS64 describes a DNS server that when asked for a domain's AAAA records, but only finds
+> A records, synthesizes the AAAA records from the A records.
+
+The synthesis in only performed if the query came in via IPv6.
+
+See [RFC 6147](https://tools.ietf.org/html/rfc6147) for more information.
+
+## Usage
+
+Translate with the well known prefix. Applies to all queries
+
+```
+dns64
+```
+
+Use a custom prefix
+
+```
+dns64 64:1337::/96
+# Or
+dns64 {
+    prefix 64:1337::/96
+}
+```
+
+Enable translation even if an existing AAAA record is present
+
+```
+dns64 {
+    translateAll
+}
+```
+
+* `prefix` specifies any local IPv6 prefix to use, instead of the well known prefix (64:ff9b::/96)
+
+## Bugs
+
+Not all features required by DNS64 are implemented, only basic AAAA synthesis.
+
+* [ ] Support "mapping of separate IPv4 ranges to separate IPv6 prefixes"
```

Comment:
these type a lists are only supported on github, just make it a normal list (or implement said features).